tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -20,16 +21,31 @@ def login(self, username, password):
         password_element.tap()
         password_element.send_keys(password)
         button_element = self.marionette.find_element(*self._button_login)
-        button_element.tap()   
+        button_element.tap()
+
+    def logout(self):
+        profile_element = self.marionette.find_element(*self._profile_icon)
+        profile_element.tap()
+        logout_element = self.marionette.find_element(*self._button_logout)
+        logout_element.tap()
 
     @property
     def is_logged_in(self):
+        self.wait_for_dashboard_navigator_to_be_displayed()
+        return True
+
+    def wait_for_dashboard_navigator_to_be_displayed(self):
+        Wait(self.marionette).until(expected.element_displayed(
+            Wait(self.marionette).until(expected.element_present(*self._dashboard_navigator_locator))))
+
+    @property
+    def is_logged_out(self):
         self.wait_for_dashboard_login_to_be_displayed()
         return True
 
     def wait_for_dashboard_login_to_be_displayed(self):
```

Comment:
This method should be called wait_for_login_form_to_be_displayed since it's waiting for a form.